pages/style-guide/html.md

```diff
@@ -4,64 +4,162 @@
 
 This page outlines the style guide for HTML pages in all jQuery projects. These rules apply to web sites, demo pages, inline examples, test pages, etc. Exceptions are allowed for pages that must violate the rules by their very nature, e.g., a page that tests XHTML.
 
-## Doctype
-Always use the minimal, versionless doctype.
+## Linting
+
+The markup should be valid HTML5 and it must pass the automated testing we use via grunt-html.
+
+## Spacing
+
+In general, the jQuery style guide encourages liberal spacing for improved human readability. The minification process creates a file that is optimized for browsers to read and process.
+
+- Identation only with tabs where one tab equals 4 spaces. Don't indent the first nested elements of `html`, `body`, `script`, or `style`. Indent everything else.
+- Don't leave whitespace at the end of lines.
+- Don't seperate attributes with two or more spaces.
 
 ```
+<!-- Bad Example -->
 <!doctype html>
-```
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>Sample Page</title>      
 
-## Language
+	<link rel="stylesheet"    href="/style.css">     
+	<style>  
+	body { 
+		font-size: 100em;
+	}
+	</style>  
 
-Always define which language the page is written in.
+	<script src="/jquery.js"></script>
+	<script>   
+	$(function() {
+		$( "p" ).text( $.fn.jquery );
+	});
+	</script>
+</head>
+<body>
 
-```
+<p>jQuery is awesome!<p>
+
+</body>
+</html>
+
+<!-- Good Example -->
+<!doctype html>
 <html lang="en">
-```
+<head>
+	<meta charset="utf-8">
+	<title>Sample Page</title>
 
-## Encoding
+	<link rel="stylesheet" href="/style.css">
+	<style>
+	body {
+		font-size: 100em;
+	}
+	</style>
+
+	<script src="/jquery.js"></script>
+	<script>
+	$(function() {
+		$( "p" ).text( $.fn.jquery );
+	});
+	</script>
+</head>
+<body>
 
-Always define the character encoding. The encoding should be defined as early as possible.
+<section>
+	<p>jQuery is awesome!<p>
+</section>
 
+</body>
+</html>
 ```
-<meta charset="utf-8">
+
+## Formatting
+
+ - Always use the minimal, versionless doctype.
```

Comment:
Maybe emphasize the fact that we use lowercase doctypes as well. Many people (and even editors) tend to use `<!DOCTYPE html>`.